lib/src/view/settings/settings_screen.dart

```diff
@@ -304,6 +310,33 @@ class _Body extends ConsumerWidget {
               );
             },
           ),
+          SettingsListTile(
+            icon: const Icon(Icons.language),
+            settingsLabel: Text(context.l10n.language),
+            settingsValue: localeL10n(context, locale),
+            onTap: () {
+              if (Theme.of(context).platform != TargetPlatform.android) {
+                showChoicePicker<Locale?>(
+                  context,
+                  choices: [
+                    null,
```

Comment:
Instead of displaying a `null` choice for the system locale, I think we should be clever and show the system locale as selected if the settings value is `null` (and also remove the duplicate in the list).